packages/testing/src/selectors.ts

```diff
@@ -1,4 +1,4 @@
-// @deprecated standart ist data-testid instead of data-testid
-export const dataTestSelector = (testId: string): string => `[data-testid="${testId}"]`
+// @deprecated standard ist data-testid instead of data-testid
+export const dataTestSelector = (testId: string): string => `[data-test-id="${testId}"]`
```

Comment:
please adapt comment "data-testid instead of data-testid", I guess one should be data-test-id

Reply:
Thanks I will adjust that